core/templates/dev/head/pages/exploration_editor/settings_tab/settings_tab.html

```diff
@@ -586,3 +593,23 @@ <h3 ng-if="action === 'publicize_exploration'">Boost exploration ranking (as mod
     <button class="btn btn-danger" ng-click="reallyDelete()">Delete Exploration</button>
   </div>
 </script>
+
+<script type="text/ng-template" id="modals/previewSummaryTile">
+  <div class="modal-header">
+    <h3>A preview for your summary tile.</h3>
+  </div>
+
+  <div class="modal-body text-center">
+    <exploration-summary-tile exploration-title="getExplorationTitle()"
+                              objective="getExplorationObjective()"
+                              category="getExplorationCategory()"
+                              thumbnail-icon-url="getThumbnailIconUrl()"
+                              thumbnail-bg-color="getThumbnailBgColor()"
+                              class="protractor-test-exp-summary-tile">
```

Comment:
Yep I was just waiting until I added the tests. I've left it in as I use it in the e2e test.